app/views/doctors/_doctor_search_form.html.haml

```diff
@@ -2,5 +2,4 @@
   = label_tag :q, t("doctor_search_form.label")
   .input-group
     = search_field_tag :q, @q, placeholder: t("doctor_search_form.placeholder")
-    = button_tag t("doctor_search_form.submit_text"), type: "submit", name: nil
-
+    = button_tag t("doctor_search_form.submit_text"), type: "submit", name: nil
```

Comment:
Since this button only works if you have JS, we should probably only show it if you do (and if you're device has location capabilities, but that's probably a bit more complex).

I recon the best way to do that is add the button into the form with JS.